t/test.lua

```diff
@@ -196,6 +196,10 @@ function issue_120(request)
   return request
 end
 
+function skipping_test(request)
+    return require 'json'.encode(request)
```

Comment:
Hm. AFAIR, we should follow strict return value structure: `{result = <...>}`. Otherwise we got nothing. See your expected result: `exp_res = {u'id': 1, u'result': [u'null']}`.

Reply:
Ok, I fixed that.  But in my opinion, the fact that now the symbolic "param" instead of null is not really something of a change.

Reply:
(I was not right, we should just return a response, it'll be wrapped into `{result = <...>} automatically.)

At least we have some evidence to say that the request is actually passed to the Lua function despite all those extra skipping steps.